apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -161,6 +161,9 @@
     {% if legacy_authz_runtime.enable is defined %}
         "legacyAuthzRuntime": {{ legacy_authz_runtime.enable }},
     {% endif %}
+    {% if console.legacy_mode is defined %}
```

Comment:
Shouldn't this come from a higher level?
Please check once the PR is ready to merge.

Reply:
This will be coming from the `identity.xml` as discussed with @chamathns